docs/manual/layout/templates/twig/_index.de.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Twig-Templates"
+description: "Übersicht Twig-Templates."
+url: "layout/templates/twig"
+aliases:
+    - /de/layout/templates/twig/
+weight: 10
+---
+
+
+{{< version "4.12" >}}
+
+Twig ist Symfony's Standardmethode zum Schreiben von Templates. Es ist schnell, sicher und leicht erweiterbar. Im Gegensatz zu 
+PHP-Templates enthalten Twig-Templates keine Geschäftslogik, was die gemeinsame Nutzung durch Designer und Programmierer erleichtert. 
+Diese Tatsache hilft, eine saubere Trennung zwischen der Präsentations- und der Daten-/Logikschicht aufrechtzuerhalten.
+
+Twig bietet außerdem viele leistungsstarke Methoden zur Strukturierung von Vorlagen, wie z. B. das Einbinden, Vererben, Wiederverwenden 
+von Blöcken oder Makros, erleichtert den Zugriff auf Objekte mit »Property Access«, verfügt über Leerzeichenkontrolle, 
+String-Interpolationsfunktionen und vieles mehr.
+
+{{% notice warning %}}
```

Comment:
I would place the warning directly after the Contao 4.12 note.

Reply:
Done